is.js

```diff
@@ -102,6 +102,14 @@ var is = function is(x) {
             }
             return currYear - parseInt(x) == THIRTEEN
         },
+        barely: {
```

Comment:
How is this different from `roughly`?

Reply:
It looks like `roughly` is either side of 13, whereas this tells us if a value is just over 13. Different usecases.

Reply:
Yeah can you also use `THIRTEEN_FUZZ` here instead of 0.2?